apps/lab/nuxt.config.ts

```diff
@@ -25,6 +25,17 @@ export default defineNuxtConfig({
     glsl: true,
   },
 
+  imports: {
+    transform: {
+      exclude: [
+        /[\/]packages[\\/]cientos[\\/]dist[\\/]trescientos\.js$/,
+        /[\/]packages[\\/]core[\\/]dist[\\/]tres\.js$/,
+        /[\/]packages[\\/]leches[\\/]dist[\\/]tresleches\.js$/,
+        /[\/]packages[\\/]postprocessing[\\/]dist[\\/]tres-post-processing\.js$/,
+      ],
+    }
+  },
+
```

Comment:
Would this be necessary for anyone using the Nuxt module or just in the monorepo? 
@userquin

Reply:
Just here, nuxt auto import plugin transforming these modules, resolving those files from the dist folder => on a real app those modules from nodes_modules and wont be transformed by nuxt.

Working at main branch since vite using aliases but auto import also transforming them.